mate-scripts/.env.sample

```diff
@@ -0,0 +1 @@
+GITHUB_TOKEN=your token
```

Comment:
does it mean students should add their tokens? Or it's just for dev purposes

Reply:
no, local script not published vie npm it is only for dev local usage